build/musicbrainz-dev/scripts/createdb.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+eval $( perl -Mlocal::lib="${MUSICBRAINZ_PERL_LOCAL_LIB}" )
+
```

Comment:
Did you miss ` set -e -u` or leave it out on purpose?

Reply:
No, it just derives from `build/musicbrainz/scripts/createdb.sh` which don’t set it either.

Note: `set -u` would potentially break here, see https://github.com/metabrainz/musicbrainz-docker/pull/123#discussion_r390221359

Reply:
Good and sad to know that.

Reply:
Rebased and followed https://github.com/metabrainz/musicbrainz-docker/commit/5e174b71238921ea045386e1730ffde02b5ea77a, see https://github.com/metabrainz/musicbrainz-docker/pull/118/commits/992af4e2db7c30247981372561f4d23358d133b1#diff-97d9c507ad8516ff5c9bc6e24001d36eR2-R7.